### PR TITLE
axgbe: upstream link issue & SFPP fix

### PR DESCRIPTION
**Pull request by swhite2**
This commit reverts the old separate patches made by me and pulls in the upstream variant in order to sync with freebsd-src. See https://github.com/opnsense/src/issues/93

### CODE REVIEW

File: sys/dev/axgbe/xgbe-phy-v2.c [hunk 1]
```diff
@@ -2890,12 +2890,10 @@ xgbe_phy_link_status(struct xgbe_prv_data *pdata, int *an_restart)
 		phy_data->rrc_count = 0;
 		if (pdata->link_workaround) {
 			ret = xgbe_phy_reset(pdata);
-			if (ret) {
+			if (ret)
 				axgbe_error("Error resetting phy\n");
-			}
-		} else {
+		} else
```
Comment (fichtner) [inline sys/dev/axgbe/xgbe-phy-v2.c:2895]: dangling else, what could possibly go wrong... some upstream style requests are just fuel to the fire :D

**Comment by fichtner** [comment]
I would just squash to get to where we are now. What do you think?

File: sys/dev/axgbe/xgbe-phy-v2.c (hunk 1 above)
Comment (swhite2) [reply sys/dev/axgbe/xgbe-phy-v2.c:2895]: I know right? Not my decision :D

**Comment by swhite2** [comment]
> I would just squash to get to where we are now. What do you think?

Seems perfectly good to me :)

File: sys/dev/axgbe/xgbe-phy-v2.c (hunk 1 above)
Comment (fichtner) [reply sys/dev/axgbe/xgbe-phy-v2.c:2895]: Yeah, no worries. Let's go with the flow.

**Comment by fichtner** [comment]
Hmm, now I cannot retain the man page in a separate commit

**Comment by fichtner** [comment]
Merged, thanks ❤️

**Pull request merged**